core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticEntity.java

```diff
@@ -14,9 +14,8 @@
     private DaoGeneticEntity() {
     }
     
-    // TODO consider renaming this class to something similar to
-    // DaoEntity because the name does not correctly capture the 
-    // new Treatment `genetic_entity` type.
+    // TODO ...something similar to DaoEntity to reflect the fact that there are also entities that
```

Comment:
replace ... with ` consider renaming this class to `

Reply:
Replaced.